algorithm/algorithm.md

```diff
@@ -0,0 +1,147 @@
+
+    1. Assign force to nodes from previous step (for Newmark integration)
+        $$ \textbf{f}_i^t = \Sigma_p N_i(\textbf{x}_p^t) m_p \textbf{a}_p^t  $$
+
+    1. Compute nodal acceleration from previous step (for Newmark integration)
+        $$ \textbf{a}_i^t = \frac{1}{m_i^t} \textbf{f}_i^{t} $$
+
+    1. Compute body force at nodes from material points
+        $$ \textbf{b}_i^{t+\Delta t} = \Sigma_p N_i(\textbf{x}_p^t) m_p G $$
+
+    1. Compute traction at nodes from material points
+        $$ \textbf{t}_i^{t+\Delta t} = \Sigma_p N_i(\textbf{x}_p^t) \frac{m_I^{t+\Delta t}}{\gamma} \frac{1}{s_p} \textbf{t}_p^{t+\Delta t} $$
+
+    1. Compute external force
+        $$ (\textbf{f}_i)^{ext,t+\Delta t} = \textbf{b}_i^{t+\Delta t} + \textbf{t}_i^{t+\Delta t} $$
+
+    1. Compute internal force
+        $$ (\textbf{f}_i)^{int,t+\Delta t} = \Sigma_i \frac{m_i^{t+\Delta t}}{\gamma} B_i(\textbf{x}_p^t) \boldsymbol{\sigma}_i^{t+\Delta t} $$
+
+    1. Compute nodal force
+        $$ \textbf{f}_i^{t+\Delta t} = (\textbf{f}_i)^{int,t+\Delta t} + (\textbf{f}_i)^{ext,t+\Delta t}  $$
+
+    1. Compute nodal acceleration
+        $$ \textbf{a}_i^{t+\Delta t} = \frac{1}{m_i^t} \textbf{f}_i^{t+\Delta t}$$
+
+    1. Compute nodal velocity
+        1. Normal Integration
+            $$ \textbf{v}_i^{t+\Delta t} = \textbf{v}_i^{t} + \Delta t \textbf{a}_i^{t+\Delta t} $$
+
+        1. Newmark Integration
+            $$ \textbf{v}_i^{t+\Delta t} = \textbf{v}_i^{t} + \Delta t (1-\gamma_N) \textbf{a}_i^t + \Delta t \gamma_N \textbf{a}_i^{t+\Delta t} $$
+
+    1. Update soil density
+        $$ \gamma = \frac{\gamma}{1 + \varepsilon_{v,p}} $$
+
+    1. Update material points acceleration
+        $$ \textbf{a}_p^{t+\Delta t} = \Sigma_i N_i(\textbf{x}_p^t) \textbf{a}_i^{t+\Delta t} $$
+
+    1. Update material points velocity
+        1. Mapping from nodal velocity
+            $$ \textbf{v}_p^{t+\Delta t} = \Sigma_i N_i(\textbf{x}_p^t) \textbf{v}_i^{t+\Delta t} $$
+
+        1. Normal Integration
+            $$ \textbf{v}_p^{t+\Delta t} = \textbf{v}_p^t + \Delta t  \textbf{a}_I^{t+\Delta t} $$
+
+        1. Newmark Integration
+            $$ \textbf{v}_p^{t+\Delta t} = \textbf{v}_p^{t} + (1-\gamma_N) \Delta t \textbf{a}_p^t + \gamma_N \Delta t \textbf{a}_p^{t+\Delta t} $$
+
+    1. Update material points position
+        1. Normal Integration
+            $$ \textbf{x}_p^{t+\Delta t} = \textbf{x}_p^t + \Delta t \textbf{v}_p^{t+\Delta t} $$
+
+        1. Newmark Integration
+            $$ \textbf{x}_p^{t+\Delta t} = \textbf{x}_p^t + \Delta t \textbf{v}_p^t + \frac{1-2\beta_N}{2} {\Delta t}^2 \textbf{a}_p^t + \beta_N {\Delta t}^2 \textbf{a}_p^{t+\Delta t} $$
+
+1. Reset the grid mesh (if it was updated) and advance to the next time step
+
+1. Generate output files (.vtk) for each sub time step  
+
+
+## Nomenclature
+
+$\textbf{a}_i^t$ acceleration of node $i$
```

Comment:
All symbols should be enclosed within `$$` and `$$`

Reply:
you don't want to do single `$ $` for inline codes? `$$ $$` is for a new line for the equation itself

Reply:
There is no `$ $` formatting in GitBooks Mathjax plugin or a distincition for the new-line equation. See my previous comment: https://github.com/cb-geo/mpm3d-doc/pull/5#discussion_r138556592

Reply:
but it's weird right? We will need inline for stuff

Reply:
I will try the other one

Reply:
It will be inline, it won't push it to a new line.